kubernetes/gke-utility/argocd/argocd-cm.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+data:
```

Comment:
We should include some comments what are different options doing and why did we enable/disable these options.